.travis.yml

```diff
@@ -1,6 +1,11 @@
 language: node_js
 node_js:
   - "stable"
+services:
+  - postgresql
 cache:
   directories:
-    - "node_modules"
```

Comment:
create an extra line at the end of the file

Reply:
I have actioned your feedback, Chris. Everything should be good now.